### PR TITLE
Style update for the Ev. Listener

### PR DESCRIPTION
**Pull request by TonyNedyalkov**

### CODE REVIEW

File: src/ecma-debugger/eventlisteners/evlisteners.css [hunk 1]
```diff
@@ -1,11 +1,6 @@
-﻿.ev-rt-list
+﻿.ev-rt-list>li
```
Comment (hzr) [inline src/ecma-debugger/eventlisteners/evlisteners.css]: Space around the combinator, so: `.ev-rt-list > li`.

File: src/ecma-debugger/eventlisteners/evlisteners.css [hunk 2]
```diff
@@ -39,17 +34,21 @@
...
 {
-  display: inline-block;
-  height: 12px;
-  line-height: 12px;
-  vertical-align: middle;
+
   margin-left: 3px;
   padding: 0 2px;
-  background-color: hsl(0, 0%, 70%);
+  background-color: rgba(0, 0, 0, 0.29);
```
Comment (hzr) [inline src/ecma-debugger/eventlisteners/evlisteners.css:40]: Why not 0.3?

File: src/ecma-debugger/eventlisteners/evlisteners.css [hunk 3]
```diff
@@ -39,17 +34,21 @@
...
   color: #fff;
   border-radius: 2px;
 }
 
+.ev-listener.tooltip-selected
+{
+  box-shadow: none;
+  color: #fff;
+  background-color: rgba(82, 122, 204, 0.553);
```
Comment (hzr) [inline src/ecma-debugger/eventlisteners/evlisteners.css]: Why not 0.55 (or 0.5 even)?

File: src/ecma-debugger/eventlisteners/evlisteners.css [hunk 4]
```diff
@@ -97,6 +96,11 @@
   margin-left: 12px;
 }
 
+.ev-rt-list .ev-rt-title
+{
+  margin-left: 0px;
```
Comment (hzr) [inline src/ecma-debugger/eventlisteners/evlisteners.css]: 0px -> 0

File: src/ecma-debugger/eventlisteners/evlisteners.css (hunk 2 above)
Comment (TonyNedyalkov) [inline src/ecma-debugger/eventlisteners/evlisteners.css:40]: Height of the badge was increased with one pixel - it's to compensate for that by making it a tiny bit lither, in order to keep the same weight of the badge.

File: src/ecma-debugger/eventlisteners/evlisteners.css (hunk 3 above)
Comment (TonyNedyalkov) [inline src/ecma-debugger/eventlisteners/evlisteners.css]: 0.55 looks good.

**Comment by danfooo** [comment]
Looks fine.

**Comment by hzr** [comment]
Yep, fine now.

**Comment by chriskr** [comment]
Tony, if two team members are OK with your changes you can pull them.

**Pull request merged**